stages/org.osbuild.firewall

```diff
@@ -72,7 +76,14 @@ def main(tree, options):
     enabled_services = options.get("enabled_services", [])
     disabled_services = options.get("disabled_services", [])
 
+    default_zone = options.get("default_zone", "")
+
     # firewall-offline-cmd does not implement --root option so we must chroot it
+    if default_zone:
+        subprocess.run(["chroot", tree, "firewall-offline-cmd", f"--set-default-zone={default_zone}"], check=True)
```

Comment:
🤷  I can try... I must admit that I just took the same approach already used in the stage without questioning it... 🤔

Reply:
I'm still in the process of testing this, but one thing I know already. Not using chroot will require installing `firewalld` in the build pipeline of all osbuild-composer distro definitions. Although we do not configure firewall in any default image definition, the user can do it via Blueprint customizations... So this could be considered a backward incompatible change if there is no fallback to keep using `chroot`.

Reply:
So the option seems to work just fine (as an equivalent of `--root` option in other tools). I think that not using `chroot` is preferred, but it turns out to be tricky at this point. I wonder what's @gicmo 's opinion...

Reply:
@teg the same question ^

IMO it makes the most sense to keep using `chroot` as it would be a backward incompatible change of behavior. Manifests which `build` pipeline does not contain `firewalld`, but they are using the `org.osbuild.firewall` stage could not be rebuilt any more.

Reply:
> IMO it makes the most sense to keep using `chroot` as it would be a backward incompatible change of behavior. Manifests which `build` pipeline does not contain `firewalld`, but they are using the `org.osbuild.firewall` stage could not be rebuilt any more.

I think for this reason alone we should keep it with the `chroot` and ignore my original comment :)

Reply:
OK, dropped the second commit and the change is back in the original state (which passed CI before, so I hope it will be the case again)...